scripts/update-preview-build-branches.sh

```diff
@@ -2,6 +2,17 @@
 git config user.name "Clarity Update Bot"
 git config user.email "noreply@github.com"
 
+# create branches for manual dispatches where the branches are not created.
+if [[ `git branch --list preview-build/$GIT_BRANCH/clr-ui` == '' ]] || [[ `git branch --list preview-build/$GIT_BRANCH/clr-angular` == '' ]]
```

Comment:
This won't work because the local branches don't exist after the checkout action checks out just the triggering branch.